src/eth1/impl/ethers.ts

```diff
@@ -89,13 +90,22 @@ export class EthersEth1Notifier extends EventEmitter implements Eth1Notifier {
     this.emit('block', block);
   }
 
-  public async processDepositLog(dataHex: string, indexHex: string): Promise<void> {
-    const deposit = this.createDeposit(dataHex, indexHex);
+  public async processDepositLog(pubkey: string, withdrawalCredentials: string, amount: string, signature: string, merkleTreeIndex: string): Promise<void> {
+    const deposit: Deposit = {
+      proof: Array.from({length: DEPOSIT_CONTRACT_TREE_DEPTH},() => Buffer.alloc(32)),
+      index: (await this.contract.from_little_endian_64(merkleTreeIndex)).toNumber(),
```

Comment:
I think you should use BN or ssz
This is breaking CI

Reply:
contract is undefined in test, it should be stubbed. we could write our own little to big endian conversion.

Reply:
Interesting, I'm getting different errors running locally. Debugging...